content/smart.md

```diff
@@ -48,6 +48,12 @@ Technically speaking, the SMART specification requires that a FHIR server exist
 
 ## Technical ##
 
+### Terra UI Framework ###
+
+Some of the things that we take into consideration when validating your SMART app are user interface (UI) and user experience (UX). They are an important part of the overall user's experience.
+
+If you are building your SMART app using [React library](https://reactjs.org/), we highly recommend that you take a look at [Terra UI framework](https://engineering.cerner.com/terra-ui/#/home/terra-ui/index) that we've created and open sourced. The UI framework follows Cerner’s Design Standard Guidelines. What this means is that your app will look and feel like a Cerner's app.
+
```

Comment:
Should we also mention : ```The framework also promotes rapid UI development.```  ?

Reply:
How about:
```
If you are building your SMART app using the [React](https://reactjs.org/) JavaScript library, we highly recommend that you take a look at the [Terra](https://engineering.cerner.com/terra-ui/#/home/terra-ui/index) UI framework that we've created and open sourced. Terra follows Cerner's Design Standard Guidelines so your app will look and feel like a Cerner app.
```

Reply:
@shriniketsarkar, We'll mention that once Kaiju is available for public use.